resources/views/reports/financial_statement/pdf/income_statement_multiple.blade.php

```diff
@@ -0,0 +1,107 @@
+
+@extends('reports.template')
+@section('page_title', 'Income Statement')
+@section('content')
+
+<table>
+    <thead>
+        <th width="50%"></th>
+        <th class="align-right">DEBIT</th>
+        <th class="align-right">CREDIT</th>    </thead>
+    <tbody>
+        <tr>
+            <td>Sales</td>
+            <td>1000</td>
+            <td>100</td>
+        </tr>
+        <tr>
+            <td>Cost of Goods Sold</td>
+            <td>1000</td>
+            <td>100</td>
+        </tr>
+    </tbody>
+    <tfoot>
+        <tr>
+            <th>Gross Profit</th>
+            <td>1000</td>
+            <td>100</td>
+        </tr>
+    </tfoot>
+
+</table>
+
+<table>
+    <thead>
+        <th class="center" width="50%">OPERATING EXPENSES</th>
+        <th>DEBIT</th>
+        <th>CREDIT</th>
+    </thead>
+    <tbody>
+        <tr>
+            <td>Salaries</td>
+            <td>1000</td>
+            <td>100</td>
+        <tr>
+        <tr>
+            <td>Rent</td>
+            <td>1000</td>
+            <td>100</td>
+        <tr>
+        <tr>
+            <td>Utilities</td>
+            <td>1000</td>
+            <td>100</td>
+        <tr>
+        <tr>
+            <td>Depreciation</td>
+            <td>1000</td>
+            <td>100</td>
+        <tr>
+    </tbody>
+    <tfoot>
+        <tr>
+            <th>Total Operating Expenses</th>
+            <td>1000</td>
+            <td>100</td>
+        </tr>
+        <tr>
+            <th>OPERATING INCOME</th>
+            <td>1000</td>
+            <td>100</td>
+        </tr>
+    </tfoot>
+</table>
+
+
+<table>
+    <thead>
+        <th class="center" width="50%">NON-OPERATING OR OTHER</th>
+        <th>DEBIT</th>
+        <th>CREDIT</th>
+    </thead>
+    <tbody>
+        <tr>
+            <td>Gains</td>
+            <td>1000</td>
+            <td>100</td>
+        <tr>
+        <tr>
+            <td>Interest Expense</td>
+            <td>1000</td>
+            <td>100</td>
+        <tr>
+    </tbody>
+    <tfoot>
+        <tr>
+            <th>Total Non-Operating or Other</th>
+            <td>1000</td>
+            <td>100</td>
+        </tr>
+        <tr>
+            <th>NET INCOME/LOSE</th>
+            <td>1000</td>
+            <td>100</td>
+        </tr>
+    </tfoot>
+</table>
+@endsection
```

Comment:
Kindly follow the format from this excel file. I made a similar change in Balance Sheet PDF so you may use it as a reference.
[Income Statement - Multiple Step.xlsx](https://github.com/justinmanigo/HMEZGEB/files/8750728/Income.Statement.-.Multiple.Step.xlsx)

The same applies also to Income Statement - Single Step.
[Income Statement - Single Step.xlsx](https://github.com/justinmanigo/HMEZGEB/files/8750734/Income.Statement.-.Single.Step.xlsx)